engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/schemaregistry/confluent/serialization/ConfluentKafkaAvroDeserializer.scala

```diff
@@ -15,22 +17,25 @@ import pl.touk.nussknacker.engine.avro.schemaregistry.confluent.client.Confluent
   *            fetching schema for deserializer has place at deploy moment. It can be happen when process has
   *            set latest version and deploy was run after new schema was added.
   */
-class ConfluentKafkaAvroDeserializer[T](schema: Schema, confluentSchemaRegistryClient: ConfluentSchemaRegistryClient, var isKey: Boolean, _useSpecificAvroReader: Boolean)
+class ConfluentKafkaAvroDeserializer[T](kafkaConfig: KafkaConfig, schema: Schema, confluentSchemaRegistryClient: ConfluentSchemaRegistryClient,
```

Comment:
I know that is not part of your changes.. But maybe we can change:

```
abstract class AbstractConfluentKafkaAvroDeserializer(useSpecificAvroReader: Boolean) 
```
I think this `_useSpecificAvroReader: Boolean` looks ugly..